backend/apps/owasp/models/award.py

```diff
@@ -0,0 +1,246 @@
+"""OWASP app award model."""
+
+from __future__ import annotations
+
+from django.db import models
+
+from apps.common.models import TimestampedModel
+
+
+class Award(TimestampedModel):
+    """OWASP Award model.
+
+    Represents OWASP awards based on the canonical source at:
+    https://github.com/OWASP/owasp.github.io/blob/main/_data/awards.yml
+    """
+
+    class Meta:
+        db_table = "owasp_awards"
+        indexes = [
+            models.Index(fields=["category", "year"], name="owasp_award_category_year"),
+            models.Index(fields=["-year"], name="owasp_award_year_desc"),
+            models.Index(fields=["name"], name="owasp_award_name"),
+        ]
+        verbose_name = "Award"
+        verbose_name_plural = "Awards"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["name", "year", "category"], name="unique_award_name_year_category"
+            )
+        ]
+
+    # Core fields based on YAML structure
+    category = models.CharField(
+        verbose_name="Category",
+        max_length=100,
+        help_text="Award category (e.g., 'WASPY', 'Lifetime Achievement')",
+    )
+    name = models.CharField(
+        verbose_name="Name",
+        max_length=200,
+        help_text="Award name/title (e.g., 'Event Person of the Year')",
+    )
+    description = models.TextField(
+        verbose_name="Description", blank=True, default="", help_text="Award description"
+    )
+    year = models.IntegerField(
+        verbose_name="Year",
+        null=True,
+        blank=True,
+        help_text="Year the award was given (null for category definitions)",
+    )
+
+    # Winner information
+    winner_name = models.CharField(
+        verbose_name="Winner Name",
+        max_length=200,
+        blank=True,
+        default="",
+        help_text="Name of the award recipient",
+    )
+    winner_info = models.TextField(
+        verbose_name="Winner Information",
+        blank=True,
+        default="",
+        help_text="Detailed information about the winner",
+    )
+    winner_image = models.CharField(
+        verbose_name="Winner Image",
+        max_length=500,
+        blank=True,
+        default="",
+        help_text="Path to winner's image",
+    )
+
+    # Award type from YAML (category or award)
+    award_type = models.CharField(
+        verbose_name="Award Type",
+        max_length=20,
+        choices=[
+            ("category", "Category"),
```

Comment:
Please also make the categories a separate subclass (see an [example](https://github.com/OWASP/Nest/blob/9df6fae00d9c787c1b97c9b9ca74f4610facdd19/backend/apps/owasp/models/snapshot.py#L14))